.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "doc/source/_static/scipy-mathjax"]
 	path = doc/source/_static/scipy-mathjax
 	url = https://github.com/scipy/scipy-mathjax.git
+[submodule "scipy/_lib/boost"]
+	path = scipy/_lib/boost
+	url = https://github.com/mckib2/boost-headers-only
```

Comment:
I seem to remember commenting on this elsewhere, but I can't find it anymore: can we move this repo under the scipy org? With that and a README that has docs on how the repo was created and how to update to a new Boost version, I think this approach is good to go.

Reply:
Right [here](https://github.com/scipy/scipy/pull/13328#issuecomment-822787507) :) I'm planning on sitting down and doing that this weekend.  I have high hopes in its automatability

Reply:
Sounds good, thanks Nicholas!

Reply:
I think we're ready to either create a new repo or transfer ownership of the existing repo to the SciPy organization.  I don't have the rights to create repos in the org and I get the following message when I try request a transfer of ownership:

    You don’t have the permission to create public repositories on scipy

I've added you, @mdhaber, and @WarrenWeckesser as collaborators to that repo.  Hopefully one of you will have the permissions to move it under the right org

Reply:
Try again, you should be able to do it now.

Reply:
Perfect!  It now exists under the SciPy org.  I noticed on the last run that the submodule didn't get updated to latest commit, so I fixed that and verified from the CI logs that the latest commit got pulled from the scipy/boost-headers-only repo, so looks good to go on this front if everything builds correctly